sass/index.scss

```diff
@@ -266,7 +266,7 @@ video::-webkit-media-controls {
             justify-content: space-between;
 
             #login {
-                @include button-primary;
+                @include button-secondary();
```

Comment:
Does this need parentheses, or can they be removed?

Reply:
doesn't need them but for the sake of not dealing with a review  im leaving them